index.js

```diff
@@ -0,0 +1,80 @@
+const addButton = document.querySelector('.button');
+const bookList = document.getElementsByClassName('book-list')[0];
+const inputTitle = document.querySelector('.title');
+const inputAuthor = document.querySelector('.author');
+const alertMessage = document.querySelector('#alert-message');
+let theBooks = [];
+
+// const storage = localStorage;
```

Comment:
**Your project is running perfectly!**

- [ ] [OPTIONAL] Pease remove unused or commented code before merging the project to the next step!